CMakeLists.txt

```diff
@@ -8,11 +8,13 @@ project(taichi)
 
 include("cmake/utils.cmake")
 
+message(WARNING "It seems that you are running cmake manually, which may cause issues. Please use setup.py to build taichi from source, see https://docs.taichi-lang.org/docs/dev_install for more details.")
```

Comment:
nit: why do we take this out of the if branch? ;) The old logic was: if we run cmake from setup.py it'll have `TI_VERSION_MAJOR` set so it won't trigger this warning, not sure if it's still applicable.

Reply:
oof it's a miss